package proto_ff;

import "yd_fieldoptions.proto";


message E_ButtonButton
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "按钮id"];
	optional int32 m_time = 2[(yd_fieldoptions.field_cname) = "读条时间"];
}


message Sheet_ButtonButton
{
	repeated E_ButtonButton E_ButtonButton_List = 1[(yd_fieldoptions.field_arysize)=64];
}
